scripts/deploy.py

```diff
@@ -142,6 +166,8 @@ def _execute_deployment():
     # Check that the current directory is correct.
     common.require_cwd_to_be_oppia()
 
+    update_cache_slug()
```

Comment:
Can this be done after stuff is copied to the release directory, but before the build step? That way, you avoid changing the files in the main oppia directory.